### ISSUE TITLE
Performance testing and sign off

### ISSUE THREAD

**Issue by zepumph** [phetsims/ratio-and-proportion#270]
From code review in https://github.com/phetsims/ratio-and-proportion/issues/210

**Comment by zepumph** [comment]
@KatieWoe could you please comment on if there were any performance-type issues that came out of initial dev testing here. I will note #267 as a potential part of that. Were there any more?

**Comment by zepumph** [comment]
When I ran the sim with ?fuzz for 12 seconds after loading and recorded the performance in chrome dev tools, here were the top hitting items:

![image](https://user-images.githubusercontent.com/6856943/103426534-a8763a80-4b66-11eb-9e23-723415054501.png)


I don't see anything that is obviously sim-specific. I did note though that >7% is taken up from recalculating the bounds of text. I think that was just because I was dragging the sim to resize as we went. Perhaps that is expected, and most likely we wouldn't encounter that in the wild. A similar thing happened in Number Play when I resized while fuzzing and recorded it, but for SVGGroup:
![image](https://user-images.githubusercontent.com/6856943/103426643-71545900-4b67-11eb-95ff-242ca3954725.png)


Since nothing has come out of dev testing involving performance. I think it will be fine to close this. @KatieWoe feel free to create new performance issues if you encounter anything in future Ratio and Proportion tests.
